pkg/parser/pipelineyml/visitor_ref_op.go

```diff
@@ -35,6 +39,7 @@ type RefOp struct {
 	Ref string // ref: alias or namespace
 	Op  string // OPERATION
 	Key string // key
+	Ex  string // Executor
```

Comment:
What is `Ex`. Add an example.

Reply:
I have add examply in test:
![image](https://user-images.githubusercontent.com/30427474/130757570-67d4fb2c-d1d4-42a6-a46c-8172971ea397.png)
![image](https://user-images.githubusercontent.com/30427474/130757723-d4114766-a801-48b2-807b-488a457e5765.png)
Ex like(quote) means help escape param